SparkPost/2web2ui
# SA-1540: UX Feedback for Report Builder

### What Changed
- [x]     Typeahead can swap out icon from search
- [x]         Timezone typeahead uses clock icon
- [x]     Increase column width of columns of bounce reasons table (and other tables in the tabs)
- [x]     Add sortable prop to group by table
- [x]     Shift the group by table by -1 px to avoid double line betweeen header (with select) and table
- [x]     Add border to legend circles in tooltip (see health score page)
- [x]     add legend circle to aggregates table at bottom
- [x]     Remove deprecated flag on count_rendered and count_initial_rendered
- [x]         Change label and description from Rendered to Opens

### How To Test
 - Verify that all tests are still valid
 - Verify that the the visual updates are in place.
